api/src/main/java/jakarta/enterprise/lang/model/AnnotationInfo.java

```diff
@@ -8,17 +8,27 @@
 /**
  * An annotation instance, typically obtained from an {@link AnnotationTarget}.
  * Provides access to annotation members and their values.
+ * <p>
+ * Implementations of this interface are required to define the {@code equals} and {@code hashCode} methods.
+ * Implementations of this interface are encouraged to define the {@code toString} method such that
+ * it returns a text resembling the corresponding Java&trade; syntax.
+ * <p>
+ * There is no guarantee that any particular annotation instance, represented by an implementation of this interface,
+ * will always be represented by the same object. That includes natural singletons such as the {@code jakarta.inject.Singleton}
+ * annotation. Instances should always be compared using {@code equals}.
+ *
+ * @since 4.0
  */
 public interface AnnotationInfo {
     /**
      * Name of the commonly used {@code value()} annotation member.
      */
-    String MEMBER_VALUE = "value";
+    String VALUE_MEMBER = "value";
```

Comment:
Maybe we should just call it `VALUE` and move it to `AnnotationMember`. `AnnotationMember.VALUE` sounds best.

Reply:
Done that.